superset-frontend/src/SqlLab/components/AceEditorWrapper/index.tsx

```diff
@@ -22,6 +22,7 @@ import { useDispatch } from 'react-redux';
 import { css, styled, usePrevious, useTheme } from '@superset-ui/core';
 import { Global } from '@emotion/react';
 
+import { SQL_EDITOR_LEFTBAR_WIDTH } from 'src/SqlLab/constants';
```

Comment:
Needn't tackle it in this PR, but I would think layout constants like these should be global (or in the theme) since they likely could/should apply to other views in the app.

Reply:
`src/SqlLab/constants.ts` looks like a good place for this constant.